test/fixtures/data-driven-tests.json

```diff
@@ -943,7 +943,7 @@
             "name": "TickSize",
             "nth_of_type": 16,
             "type": 16,
-            "expected_hex": "01010"
+            "expected_hex": "001010"
```

Comment:
This test was wrong... The binary parser reads the first set of bytes, and should've caused an error, unsure as to why the old code was reading this as `001010` and not `010100`

Reply:
This is really odd. 

Did this test run and pass before?

Reply:
Yes, but it should not have, and I'm not exactly sure why it did.

Reply:
Turns out we are getting some collisions with the way that field.ordinal is calculated. Will explain on a call either today or monday.